stores/moderationSubmissionsStore.ts

```diff
@@ -4,17 +4,41 @@ import { ref, type Ref } from 'vue'
 import { gqlClient } from '../utils/graphql.js'
 import { type Submission } from '~/typedefs/gqlTypes.js'
 
+export enum SelectedSubmissionListViewTab {
+    forReview = 'forReview',
+    approved = 'approved',
+    rejected = 'rejected'
+}
+
 export const useModerationSubmissionsStore = defineStore(
     'submissionsStore',
     () => {
         const submissionsData: Ref<Submission[]> = ref([])
+        const filteredSubmissionDataForListComponent: Ref<Submission[]> = ref([])
 
         async function getSubmissions() {
             const submissionsSearchResults = await querySubmissions()
             submissionsData.value = submissionsSearchResults
         }
 
-        return { getSubmissions, submissionsData }
+        function filterSubmissionsBySelectedTab(tabValue: SelectedSubmissionListViewTab) {
+            switch (tabValue) {
+                case 'forReview':
```

Comment:
I think you can use `SelectedSubmissionListViewTab.forReview`, etc instead of the strings here.

Reply:
+1 let's avoid hardcoded strings and use Enums (looks like you already have one)